src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -152,14 +152,20 @@ The `id` property allows you to explicitly define the identifier used for your a
 
 #### `start_url` {: #start-url }
 
-The `start_url` is required and tells the browser where your application
+The `start_url` is optional and defaults to document URL. It tells the browser where your application
```

Comment:
Source: https://developer.mozilla.org/en-US/docs/Web/Manifest/start_url

![Screenshot 2023-02-21 at 15 05 44](https://user-images.githubusercontent.com/4297022/220367221-649189b1-2ce8-4077-b166-3aa64e8d064d.png)

Reply:
In this context, the fields are not explained for their role in the manifest spec, but for their role in PWA. To consider an app installable, browsers still check for the existence of the start_url, in that sense is that the field si required. The change in this PR is not correct for the context.